components/brave_vpn/brave_vpn_service_desktop.cc

```diff
@@ -258,3 +370,7 @@ void BraveVpnServiceDesktop::GetAllRegions(GetAllRegionsCallback callback) {
   }
   std::move(callback).Run(std::move(regions));
 }
+
+void BraveVpnServiceDesktop::GetDeviceRegion(GetDeviceRegionCallback callback) {
+  std::move(callback).Run(device_region_.Clone());
```

Comment:
why do we clone here?

Reply:
`using GetDeviceRegionCallback = base::OnceCallback<void(RegionPtr)>;`
Callback accepts `RegionPtr` as an argument. it's cloned data from `Region` struct.
To pass args via mojo message, it seems copied data should be passed.